src/Symfony/Component/Config/Tests/Resource/DirectoryResourceTest.php

```diff
@@ -96,8 +96,10 @@ public function testIsFreshNewFileWithDifferentPattern()
     public function testIsFreshDeleteFile()
     {
         $resource = new DirectoryResource($this->directory);
+        $time = time();
+        sleep(1);
         unlink($this->directory.'/tmp.xml');
-        $this->assertFalse($resource->isFresh(time()), '->isFresh() returns false if an existing file is removed');
+        $this->assertFalse($resource->isFresh($time), '->isFresh() returns false if an existing file is removed');
```

Comment:
@xabbuh We can't use that because the internal implementation only relates to the PHP interpreter. This relies on the *filesystem* and is external of the interpreter.

> An added bonus of using the ClockMock class is that time passes instantly. Using PHP's sleep(10) will make your test wait for 10 actual seconds (more or less). In contrast, the ClockMock class advances the internal clock the given number of seconds without actually waiting that time, so your test will execute 10 seconds faster.
- http://symfony.com/blog/new-in-symfony-2-8-clock-mocking-and-time-sensitive-tests